src/app/repository/local-repository.service.ts

```diff
@@ -145,6 +145,9 @@ export class LocalRepositoryService {
                 return Promise.resolve();
             }
 
+            // Expand added folders
+            this.setFolderExpansion(missing.concat("local"), true);
```

Comment:
This should be moved to a place where `addLocalFolders` is used, because this way you can't opt-out of the expansion behavior.